src/components/Table/ImageTable.js

```diff
@@ -170,9 +170,9 @@ ImageRow.propTypes = {
 const ImageTable = (props) => {
   const intl = useIntl();
   // start sorting by created date
-  const [activeSortIndex, setActiveSortIndex] = useState(null);
-  const [isSortAscending, setIsSortAscending] = useState(true);
-  const [sortBy, setSortBy] = useState("id");
+  const [activeSortIndex, setActiveSortIndex] = useState(3);
+  const [isSortAscending, setIsSortAscending] = useState(false);
+  const [sortBy, setSortBy] = useState(3);
```

Comment:
This should still be "job_created". The setup is a little weird here but basically... the `activeSortIndex` is used by the table headers for UI purposes, `isSorbtAscending` is used in both the headers and the sorting function, `sortBy` is used only in the sorting function.

Reply:
Oh ok, it's a bit confusing, but I think I understand it now. Let's give the review one more try